dev/devicelab/lib/tasks/microbenchmarks.dart

```diff
@@ -63,7 +63,7 @@ TaskFunction createMicrobenchmarkTask() {
   };
 }
 
-Future<Process> _startFlutter({
+Future<Process> startFlutter({
```

Comment:
Can we just move these to something under `dev/devicelab/lib` that's common? Alternatively, let's make the task creation function in this file?

It just seems strange to make these public. They're very implementation dependent, and someone will probably import them incorrectly later, or unintentionally break them for the current use cases.

Reply:
+1 to this

Reply:
done